Release 2.27.1 (#9858)
timescale/timescaledb: CHANGELOG.md

you use psql to update your database, use the -X flag and prevent any .psqlrc
commands from accidentally triggering the load of a previous DB version.**

## 2.27.1 (2026-05-19)

> This release contains performance improvements and bug fixes since the 2.27.0 release. We recommend that you upgrade at the next available opportunity.

Reviewer: ```suggestion
This release contains bug fixes for the 2.27.0 release. We recommend that you upgrade at the next available opportunity.
```
